### DISCUSSION TITLE
Registration Process

### DISCUSSION THREAD

**Discussion by simple-AMS** [YetiForceCompany/YetiForceCRM#15071] [General] [answered]
Just curious if the system is free of charge, why bother with a registration process in the first place? It seems that registrations take forever to get approved and every time you login, you're met with a nuisance message stating that the system hasn't been registered yet it has said Registration Successful countless times and the status never changes from pending approval. Makes no sense!

**Comment by bpabiszczak** [comment, marked as answer]
The system is free of charge, but the license requires real data to register the system.
Acceptance of registration is manual, because if it was not, then 70% of registrations were false [false data was provided], currently only 5% are false.

**Reply by simple-AMS** [reply]
Makes sense. Real data was used for a client I was assisting, yet the status never changes. Has said Pending Approval for days. Why would it not change if real data was used?

**Reply by bpabiszczak** [reply]
We accept no more than 48 hours, currently all of them are accepted or rejected [use the button check status in your system].

**Reply by simple-AMS** [reply]
Cool, thank you sir :)

**Reply by mariuszkrzaczkowski** [reply]
If the registration status does not change, it is necessary to check the server configuration because there is often a configuration problem, which makes communication with our server impossible. It is worth checking all the logs: WWW server, PHP, FPM

**Reply by simple-AMS** [reply]
@bpabiszczak What is the best way to schedule a quick meeting with you?

**Reply by bpabiszczak** [reply]
I do not know English well :( How can I help you?

**Reply by simple-AMS** [reply]
@mariuszkrzaczkowski Thanks Mariusz :)

**Reply by simple-AMS** [reply]
@bpabiszczak Years ago, you had some of your people help us with a project, they were pretty decent with the English. I don't want to say on here, maybe email or Skype, but wanted to run something by you and get your thoughts.
